### PR TITLE
Add ACT TAC requirements and README info

### PR DESCRIPTION
**Pull request by rnjudge**
This commit adds information about the requirements to join ACT and ACT
TAC at various stages of a project (i.e. incubating or affiliate). It
also adds info to the README about the current projects that are a part
of ACT.

Signed-off-by: Rose Judge <rjudge@vmware.com>

### CODE REVIEW

**Comment by rnjudge** [comment]
@kestewart Please review (I'm not able to add reviewers). Will also send to the ACT TAC mailing list for approval.

File: TAC-Requirements.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Requirements to join ACT
+The following document outlines requirements for projects interested in joining the Automating Compliance Tooling Project umbrella. All ACT umbrella projects are required to participate in bi-monthly Technical Advisory Committee (TAC) calls. ACT projects and affiliate projects maintain one vote in all TAC voting body decisions.
+
+## General Criteria for new projects to join the ACT umbrella
+* Willingness to generate/consume/interoperate with [SPDX](https://spdx.dev/).
+* Formal governance in place meeting Linux Foundation governance requirements.
```
Comment (rnjudge) [inline TAC-Requirements.md]: @kestewart is there a reference link available for this?

File: TAC-Requirements.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Incubating project maintainers will participate in a TAC meeting for discussion and Q&A. Any resulting action items are addressed.
+* The project has multiple participants and code development beyond original contribution.
+* Formal governance in place meeting Linux Foundation governance requirements.
+* TAC vote to accept out of incubating status through a two-thirds vote of the TAC.
+* Sponsor has verified that the project has sufficiently demonstrated the project to be usable and substantial.
+* The project demonstrates CII best practices (vulnerability handling, etc.) and maintains a passing criteria.
+* Project has a code of conduct and has demonstrated it is a welcoming environment for contributors.
+
+# Revocation Policy
+* When any of the general criteria no longer applies, the project will be given a warning. After a 6 month warning of inaction on behalf of the project,  the TAC will hold a vote (requiring two thirds consensus) to remove the project.
```
Comment (mcjaeger) [inline TAC-Requirements.md]: proposal is to specify from what it is removed (incubating status, affiliate status, or both)

File: TAC-Requirements.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* Support for SPDX.
+    * If SBOM producing tool, assumption is that tool produces valid SPDX documents
+    * If consumption tool, assumption is that tool is able to consume valid SPDX documents
+* Data source neutrality must be possible.
+* Project/tool is established and fully functional (not proof of concepts).
+* The software for the project must be licensed under an OSI approved license.
+* Presentation to TAC members showing how the tool works amd its overall usefulness/completeness.
+* At least one ACT project member endorsement, and willing to “sponsor”.
+* TAC members may not sponsor a project for which they have a clear conflict of interest (for example, originating primarily from their organization). This doesn’t mean that they can’t have any involvement at all - for example, contributing pull requests, or being an end user of that project, can signal a healthy interest in and knowledge of a worthwhile project.
+* TAC vote to accept through a two-thirds vote of the TAC and liaison.
```
Comment (mcjaeger) [inline TAC-Requirements.md]: I wondered what does "and liason" mean here?"

**Review by mcjaeger — COMMENTED** [review]
just two question to maybe improve readability of the document.

File: TAC-Requirements.md (hunk 1 above)
Comment (kestewart) [reply TAC-Requirements.md]: How about rephrasing this as:
* Formal project technical charter in place approved by the Linux Foundation.   

There is no real "one size" I can point to.

File: TAC-Requirements.md (hunk 3 above)
Comment (kestewart) [reply TAC-Requirements.md]: should it be "raised by liason"?

File: TAC-Requirements.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* Project has a code of conduct and has demonstrated it is a welcoming environment.
+
+## Criteria to Graduate from Incubating to TAC Affiliate Project/voting member
+* Project has been in incubating stage at least 6 months
+* Incubating project maintainers will participate in a TAC meeting for discussion and Q&A. Any resulting action items are addressed.
+* The project has multiple participants and code development beyond original contribution.
+* Formal governance in place meeting Linux Foundation governance requirements.
```
Comment (rnjudge) [inline TAC-Requirements.md:44]: Formal project technical charter in place approved by the Linux Foundation.

File: TAC-Requirements.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* Data source neutrality must be possible.
+* Project/tool is established and fully functional (not proof of concepts).
+* The software for the project must be licensed under an OSI approved license.
+* Presentation to TAC members showing how the tool works amd its overall usefulness/completeness.
+* At least one ACT project member endorsement, and willing to “sponsor”.
+* TAC members may not sponsor a project for which they have a clear conflict of interest (for example, originating primarily from their organization). This doesn’t mean that they can’t have any involvement at all - for example, contributing pull requests, or being an end user of that project, can signal a healthy interest in and knowledge of a worthwhile project.
+* TAC vote to accept through a two-thirds vote of the TAC and liaison.
+* Project has a code of conduct and has demonstrated it is a welcoming environment.
+
+## Criteria to Graduate from Incubating to TAC Affiliate Project/voting member
```
Comment (rnjudge) [inline TAC-Requirements.md]: Incubating to become a regular project.

File: TAC-Requirements.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+# Requirements to join ACT
+The following document outlines requirements for projects interested in joining the Automating Compliance Tooling Project umbrella. All ACT umbrella projects are required to participate in bi-monthly Technical Advisory Committee (TAC) calls. ACT projects and affiliate projects maintain one vote in all TAC voting body decisions.
+
+## General Criteria for new projects to join the ACT umbrella
```
Comment (rnjudge) [inline TAC-Requirements.md]: new to "non affiliate"

File: TAC-Requirements.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    * If consumption tool, assumption is that tool is able to consume valid SPDX documents
+* Formal governance in place meeting Linux Foundation governance requirements.
+* "Proof of concept" tools  are ok i.e. not fully functional tools that might aim to solve a larger ecosystem problem.
+* The software for the project must be licensed under an OSI approved license.
+* Data source neutrality must be possible.
+* Source code must be publicly available.
+* How to contribute to the project is documented.
+* Prior to being an incubation project, a presentation will be made to TAC members discussing the general usefulness of the tool and demonstrating how it works as well as any other relevant information (project roadmap, etc).
+* One ACT TAC project member must be willing to “sponsor” and act as a liason on behalf of the incubating project.
+* Note: Incubating projects do not receive an ACT TAC vote until they have graduated to affiliate project status.
```
Comment (kestewart) [inline TAC-Requirements.md]: s/incubating/regular/

File: README.md [hunk 8]
```diff
@@ -1,2 +1,20 @@
...
+Our mission is to support development of open source tooling for efficient and effective exchange of software bill of materials (SBOM) to enable license compliance, security, export control, pedigree and provenance workflows.
+
+## Current Projects under the ACT umbrella
+* [FOSSology](https://www.fossology.org/)
+* [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort)
+* [SPDX Tools](https://github.com/spdx/tools)
+* [Tern](https://github.com/tern-tools/tern)
+* [QMSTR](https://qmstr.org/)
+
+## ACT Affiliate Projects
```
Comment (rnjudge) [inline README.md:14]: Affiliate projects are ACT projects not under Linux Foundation project governance or charters.

File: README.md [hunk 9]
```diff
@@ -1,2 +1,20 @@
...
+* [ScanCode](https://github.com/nexB/scancode-toolkit)
+* [REUSE](https://reuse.software/)
+
+## ACT Incubating Projects
```
Comment (rnjudge) [inline README.md:21]: Incubating to become regular projects

File: README.md [hunk 10]
```diff
@@ -1,2 +1,20 @@
-# TAC
-Automating Compliance Tooling Project - Technical Advisory Council
+# [Automating Compliance Tooling (ACT)](https://automatecompliance.org/)
+Our mission is to support development of open source tooling for efficient and effective exchange of software bill of materials (SBOM) to enable license compliance, security, export control, pedigree and provenance workflows.
+
+## Current Projects under the ACT umbrella
```
Comment (rnjudge) [inline README.md:4]: Under Linux Foundation project governance or charters.

**Review by kestewart — APPROVED** [review]

**Pull request merged**